inst/include/child_model_simulation.hpp

```diff
@@ -520,124 +526,116 @@ void run_bf_transmission_rate(int t,
-              (pow(1 - cpars.PMTCT_dropout(4,t) * 2, 5))  * (pow(1 - cpars.PMTCT_dropout(5,t) * 2, bf-5));
+          i_hc.percent_on_treatment = i_hc.PMTCT_coverage(hp) *
+                                      pow(1 - p_hc.PMTCT_dropout(4, t) * 2, 5) *
+                                      pow(1 - p_hc.PMTCT_dropout(5, t) * 2, bf - 5);
         }
-        intermediate.children.percent_on_treatment -= intermediate.children.previous_mtct(hp);
-        intermediate.children.previous_mtct(hp) += intermediate.children.percent_on_treatment *
-          2 * (1 - cpars.breastfeeding_duration_art(bf, t)) * cpars.PMTCT_transmission_rate(4,hp,1);
-
-        intermediate.children.bf_transmission_rate(index) += intermediate.children.percent_on_treatment *
-            2 * (1 - cpars.breastfeeding_duration_art(bf, t)) * cpars.PMTCT_transmission_rate(4,hp,1);
-        intermediate.children.percent_no_treatment -= intermediate.children.percent_on_treatment;
+        i_hc.percent_on_treatment -= i_hc.previous_mtct(hp);
+        i_hc.previous_mtct(hp) += i_hc.percent_on_treatment *
+                                  p_hc.PMTCT_transmission_rate(4,hp,1) *
+                                  2 * (1 - p_hc.breastfeeding_duration_art(bf, t));
+        i_hc.bf_transmission_rate(index) += i_hc.percent_on_treatment *
+                                            p_hc.PMTCT_transmission_rate(4, hp, 1) *
+                                            2 * (1 - p_hc.breastfeeding_duration_art(bf, t));
+        i_hc.percent_no_treatment -= i_hc.percent_on_treatment;
       }
-
     }
-    if (intermediate.children.percent_no_treatment < 0) {
-      intermediate.children.percent_no_treatment = 0;
-    }
-
-    //No treatment
-    if (cpars.breastfeeding_duration_no_art(bf, t) < 1) {
-      intermediate.children.bf_transmission_rate(index) +=
-        intermediate.children.percent_no_treatment * (1 - cpars.breastfeeding_duration_no_art(bf, t)) *
-        (2 * intermediate.children.prop_wlhiv_lt200 * cpars.vertical_transmission_rate(4,1) +
-        2 * intermediate.children.prop_wlhiv_200to350 * cpars.vertical_transmission_rate(2,1) +
-        2 * intermediate.children.prop_wlhiv_gte350 * cpars.vertical_transmission_rate(0,1));
+  
+    i_hc.percent_no_treatment = std::max(i_hc.percent_no_treatment, 0.0);
+
+    // No treatment
+    if (p_hc.breastfeeding_duration_no_art(bf, t) < 1) {
+      auto vertical_transmission = i_hc.prop_wlhiv_lt200 * p_hc.vertical_transmission_rate(4, 1) +
+                                   i_hc.prop_wlhiv_200to350 * p_hc.vertical_transmission_rate(2,1) +
+                                   i_hc.prop_wlhiv_gte350 * p_hc.vertical_transmission_rate(0,1);
+      i_hc.bf_transmission_rate(index) += i_hc.percent_no_treatment *
+                                          vertical_transmission *
+                                          2 * (1 - p_hc.breastfeeding_duration_no_art(bf, t));
```

Comment:
pulled out a factor of 2 here and defined the rest in the brackets as `vertical_transmission`